### PR TITLE
Fix 'react-unknown-prop'

### PR DESCRIPTION
**Pull request by psdcoder**
Fix to #112. Add blacklist of props which shouldn't pass to textarea.

### CODE REVIEW

**Comment by richcsmith** [comment]
@andreypopp do you think we can get this merged?

**Comment by psdcoder** [comment]
Hi, what status of PR?)

File: src/TextareaAutosize.js [hunk 1]
```diff
@@ -92,9 +93,18 @@ export default class TextareaAutosize extends React.Component {
     if (maxHeight < this.state.height) {
       props.style.overflow = 'hidden';
     }
+
+    const filteredProps = Object.keys(props).reduce((acc, prop) => {
+      if (blacklistProps.indexOf(prop) === -1) {
```
Comment (hozefaj) [inline src/TextareaAutosize.js]: Facebook recommends a somewhat cleaner solution https://facebook.github.io/react/warnings/unknown-prop.html

Comment (psdcoder) [inline src/TextareaAutosize.js]: I think filtering better than deleting all unnecessary props from cloned object, it's more readable.
And there is only one right solution, isn't it?)

**Comment by jdalton** [comment]
Let's go with the recommended approach.

**Pull request closed without merging**